block_test.go

```diff
@@ -44,7 +46,9 @@ func TestBlockMetaMustNeverBeVersion2(t *testing.T) {
 func TestSetCompactionFailed(t *testing.T) {
 	tmpdir, err := ioutil.TempDir("", "test")
 	testutil.Ok(t, err)
-	defer os.RemoveAll(tmpdir)
+	defer func() {
+		testutil.Ok(t, os.RemoveAll(tmpdir))
```

Comment:
You can inline all of those without newlines like:

```
defer func() { testutil.Ok(t, os.RemoveAll(tmpdir)) } 
```
Formatter should allow you do so.

Reply:
yes I tried it but personally like the like the one with more lines more. 

If we get one more vote to make it a one liner will update.

Reply:
:+1: